packages/core/admin/admin/src/content-manager/components/DragLayer/RelationDragPreview.js

```diff
@@ -43,12 +43,14 @@ export const RelationDragPreview = ({ status, displayedValue, width }) => {
           <IconButton noBorder>
             <Drag />
           </IconButton>
-          <ChildrenWrapper justifyContent="space-between">
-            <LinkEllipsis minWidth={0} paddingTop={1} paddingBottom={1} paddingRight={4}>
-              <Typography textColor="primary600" ellipsis>
-                {displayedValue}
-              </Typography>
-            </LinkEllipsis>
+          <ChildrenWrapper maxWidth="100%" justifyContent="space-between">
+            <Box minWidth={0} paddingTop={1} paddingBottom={1} paddingRight={4}>
```

Comment:
I think you have the ellipsis fix on this branch too 👀

Reply:
Yeah i branched off the branch -> -> -> branches